helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl

```diff
@@ -12,6 +12,8 @@ spec:
       kubeadmConfigSpec:
         clusterConfiguration:
           apiServer:
+            certSANs:
+              - "api.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}"
```

Comment:
FMI: what's the difference between `{{ include "resource.default.name" $ }}` and `{{ include "resource.default.name" . }}` ?

Reply:
> FMI: what's the difference between `{{ include "resource.default.name" $ }}` and `{{ include "resource.default.name" . }}` ?

`.` is the current scope (like if you use `{{ with ... }}`). `$` is the root scope which is the same everywhere.